### PR TITLE
pyfive recipe

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

File: recipes/pyfive/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.0" %}
+{% set sha256 = "d96eafa5c80e06125a01f36b10406acf1e6d16493e7a1e67066cf4d1b631f193" %}
+
+package:
+  name: pyfive
+  version: {{ version }}
+
+source:
+  fn: simplejson-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/pyfive/meta.yaml]: `simplejson` -> `pyfive`

**Comment by jakirkham** [comment]
One minor comment. Otherwise looks fine to me.

**Comment by jakirkham** [comment]
That's fun. Guessing this is another `conda-build-all` error (picking up the other recipe). See the [log]( https://travis-ci.org/conda-forge/staged-recipes/jobs/147342205#L1236 ).

```
+ python setup.py install --single-version-externally-managed --record record.txt

python: can't open file 'setup.py': [Errno 2] No such file or directory
```

**Comment by jjhelmus** [comment]
Strange.  Have to head out in a few minutes but will check back in a few hours.

**Comment by jakirkham** [comment]
I restarted them now that `branca` has cleared. Am expecting they will pass without issues and then will merge (if that's ok).

**Comment by jakirkham** [comment]
Should note that as this is restarted the old Travis CI log is gone. However, the old AppVeyor [log]( https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.5017/job/ohy533sqklnyi42w ) tells the same tale. Should also note that with a lower version of `conda-build` (as is pinned on Windows Python 3.4 64-bit), the issue is absent. This further indicates it was indeed a `conda-build-all` problem.

**Comment by jakirkham** [comment]
Yep, passes. Definitely a `conda-build-all` issue.

LGTM

**Comment by jakirkham** [comment]
Thanks @jjhelmus.

**Comment by jakirkham** [comment]
Raised this `conda-build-all` issue ( https://github.com/SciTools/conda-build-all/issues/52 ) upstream.

**Comment by jjhelmus** [comment]
Thanks @jakirkham for keeping an eye on this one.  Glad to have packages for this up on `conda-forge`

**Pull request merged**